vllm_ascend/quantization/w8a8_dynamic.py

```diff
@@ -703,11 +703,17 @@ def apply(
 
         topk_weights = topk_weights.to(x.dtype)
 
+        # When calculating topk, we prefetch the weights of w1 when using torchair graph, test for eager model no performance gain.
+        # Additionally, since the L2 cache is only about a little over 100MB, we set it to 80MB to prevent memory overflow.
```

Comment:
It seems this L2 cache is hardware related, is there any different on A2 or A3?

Reply:
L2 cache can work on A2 and A3